packages/wrangler/src/deploy/deploy.ts

```diff
@@ -588,7 +589,15 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 				})
 			: undefined;
 
-		const assets = await syncAssets(
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		const _assetManifest = await syncAssets(
+			accountId,
+			scriptName,
+			props.experimentalAssets,
```

Comment:
are we not passing the whole `ExperimentalAssets` object here? I'm assuming `experimental_assets.binding` will be needed at some point

Reply:
I don't think the binding will be needed at this point? this particular function just uploads assets so all it needs is the assets directory as far as I can tell

Reply:
right. What I meant was, shouldn't `props.experimentalAssets` be of type `ExperimentalAssets`? in which case we'd need to pass `props.experimentalAssets.directory` here

Reply:
I'm fairly sure the binding is not needed at the point, but also just realised the ExperimentalAssets object was not properly passed in so will fix that